pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -248,6 +248,31 @@ func fixMachineDriverLibvirtInstalled() (bool, error) {
 	return true, nil
 }
 
+func checkOldMachineDriverLibvirtInstalled() (bool, error) {
+	logging.Debugf("Checking if an older libvirt driver %s is installed", libvirtDriverCommand)
+	oldLibvirtDriverPath := filepath.Join("/usr/local/bin/", libvirtDriverCommand)
+	if _, err := os.Stat(oldLibvirtDriverPath); !os.IsNotExist(err) {
+		return false, fmt.Errorf("Found old system-wide crc-machine-driver binary")
+	}
+	logging.Debugf("No older %s installation found", libvirtDriverCommand)
+
+	return true, nil
+}
+
+func fixOldMachineDriverLibvirtInstalled() (bool, error) {
+	oldLibvirtDriverPath := filepath.Join("/usr/local/bin/", libvirtDriverCommand)
+	logging.Debugf("Removing %s", oldLibvirtDriverPath)
+	_, _, err := crcos.RunWithPrivilege("rm", oldLibvirtDriverPath)
```

Comment:
@cfergeau shouldn't it `rm -f` otherwise it might prompt user for provide `Y/N` option?

Reply:
I had no such issue during my testing, but it will be better with -f, I'll add it ;)